src/screens/NationDetailsContainer/NationDetailsScreen/index.js

```diff
@@ -55,13 +68,33 @@ class NationDetailsScreen extends Component {
     );
   }
 
+  // THIS NEEDS FIXING!!!
+  // MESSAGEVIEW SHOULD FORMAT AND PRESENT THE PANEL INFORMATION ALL BY ITSELF, WITH ALL THE FORMATTING BUILT INTO IT.
+  // WE CAN PASS IT CHUNKS OF STUFF, TEXT AND TITLES AND SUCH.
+
+// MessageView Props: title = text, messageText = text, style, renderBottom = method, renderAdditionalInfo = method, children = obj
   _buildAboutView(nation) {
+    return (
+      <MessageView style={styles.messageView} title={`About ${nation.name}...`} >
+        {/* Children are shown at the top of the panel */}
+        <Text style={styles.panelBody}>
+          Ethereum Address:
+          {nation.ethAddress}
+        </Text>
+    </MessageView>
+    );
+  }
+
+
+  // THIS IS NOT THE RIGHT WAY TO BUILD THESE WITH MESSAGE VIEW!
```

Comment:
Fix it or add "@todo"

Reply:
And an issue

Reply:
Albert and I are working on this exact code, so we will fix as we go. No issue needed, I'm just adding notes internal to the code.